radixdlt-java/src/main/java/com/radixdlt/client/application/translate/tokens/TokenUnitConvert.java

```diff
@@ -0,0 +1,128 @@
+package com.radixdlt.client.application.translate.tokens;
+
+import java.math.BigDecimal;
+import java.math.BigInteger;
+import org.radix.utils.UInt256;
+import org.radix.utils.UInt256s;
+
+/**
+ * Utility class for converting token units between UInt256 and BigDecimal
+ */
+public final class TokenUnitConvert {
```

Comment:
Should probably be called `TokenUnitConversions`.  Would then be called as `TokenUnitConversions.subunitsToUnits(foo)` which reads better IMO.

Otherwise `ConvertTokenUnits` would be fine too, but has the disadvantage that it doesn't start with `Token`.